.travis.yml

```diff
@@ -14,6 +14,9 @@ before_install:
   - sudo apt-get -qq update
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+  - pip install -U pip setuptools virtualenvwrapper
+  - source $(which virtualenvwrapper.sh)
+  - wipeenv
```

Comment:
Line 17-19 can be replaced with:
```
  - pip freeze | grep -vw "pip" | xargs pip uninstall -y
  - pip install --upgrade pip
```
This will remove all `pip`-preinstalled packages in Travis' virtual environment (`attrs`, `mock`, `nose`, `numpy`, `pbr`, `pluggy`, `py`, `pytest` and `six`), except `pip` itself. Then upgrade `pip` to latest version (ase before).